### ISSUE TITLE
00_pytorch_fundamentals_exercises Google colab

### ISSUE THREAD

**Issue by rjna** [DareData/lp-pytorch-fundamentals#7]
Google colab exercises need permission to be seen. How it be possible to give anyone with the link rights to see?

**Comment by rjna** [comment]
@ivopbernardo FYI

**Comment by ivopbernardo** [comment]
makes sense - @Gustavo-SF when possible, can you share the colab file with everyone (permissions are for every daredata account but as this is open source, we can open it to anyone in the link)
